bin/completion-handlers.ts

```diff
@@ -1,6 +1,8 @@
 // TODO: i do not see any completion functionality in this file. nothing is being provided for the defined commands of these package managers. this is a blocker for release. every each of them should be handled.
 import { Completion } from '../src/index.js';
 
+const noopCompletion = async () => [];
```

Comment:
this is not an efficiency and performance win, it's more like a syntactic optimization.